src/views/Home.vue

```diff
@@ -1,68 +1,132 @@
 <template>
   <ion-page>
-    <ion-header :translucent="true">
-      <ion-toolbar>
-        <ion-title>Blank</ion-title>
-      </ion-toolbar>
-    </ion-header>
-    
+  <ion-toolbar>
+    <ion-button id="signInButton" button @click="buttonClickSignIn()">{{user.firstName}}</ion-button>
+    <h1>LOGO</h1>
+    <ion-title>Purchase Order</ion-title>
+  </ion-toolbar>
     <ion-content :fullscreen="true">
-      <ion-header collapse="condense">
-        <ion-toolbar>
-          <ion-title size="large">Blank</ion-title>
-        </ion-toolbar>
-      </ion-header>
-    
-      <div id="container">
-        <strong>Ready to create an app?</strong>
-        <p>Start with Ionic <a target="_blank" rel="noopener noreferrer" href="https://ionicframework.com/docs/components">UI Components</a></p>
+      <div class="filterButton">
+        <ion-button button @click="handleClick(PurchaseStatus.PENDING)">Pending</ion-button>
+        <ion-button button @click="handleClick(PurchaseStatus.APPROVED)">Approve</ion-button>
+        <ion-button button @click="handleClick(PurchaseStatus.DECLINED)">Declined</ion-button>
+        <ion-button button @click="handleClick(PurchaseStatus.CLOSE)">Closed</ion-button>
       </div>
+      <ion-grid class="tableTitle">
+        <ion-row>
+          <ion-col> PO Number </ion-col>
+          <ion-col> PO Description </ion-col>
+          <ion-col> Request Date </ion-col>
+          <ion-col> Status </ion-col>
+          <ion-col> Action </ion-col>
+        </ion-row>
+      </ion-grid>
+      <order-list :orders = "orders" :status= "status"/>
+      <list-jerry :user = "user"/>
+      <ion-button id="addButton" button @click="buttonClickAddOrder()">add Icon</ion-button>
     </ion-content>
   </ion-page>
 </template>
 
 <script lang="ts">
-import { IonContent, IonHeader, IonPage, IonTitle, IonToolbar } from '@ionic/vue';
-import { defineComponent } from 'vue';
+import PurchaseAction from '@/core/enums/action.enum';
+import PurchaseStatus from '@/core/enums/status.enum';
+import { IonContent, IonPage, IonTitle, IonToolbar, IonGrid, IonRow, IonButton } from '@ionic/vue';
+import { defineComponent, ref } from 'vue';
+import IPurchaseOrder from '../core/interfaces/purchase-order';
+import IUser from '../core/interfaces/user';
```

Comment:
Use complete path to avoid this scenario ../../../../core/interfaces

Reply:
done